README.md

```diff
@@ -310,7 +310,7 @@ describe("Test firestore", () => {
 
    // Emulate Firestore
    if (shouldUseEmulator) {
-     firestoreSettings.host = "localhost:8080";
+     firestoreSettings.host = "localhost:_8080_";
```

Comment:
Same here

Reply:
Ditto.

Reply:
(BTW: when I installed the firebase emulator, I had to go with non-default ports.  So I went with :6000, :6001, :6002, ...)